ISA.md

```diff
@@ -54,7 +54,7 @@ The 'name' is for either a group or single instruction.
 Bit Mask  | Name | Count | Description
 ----------|------|-------|------------
 0000 XXXX |      |    16 | Reserved
-0001 0XXX | JUMP |     8 | Jump, see section below
+0001 0FFF | JUMP |     8 | Jump, see section below
```

Comment:
Don't like this change. `X` is standard way of saying you don't really care about the state of this bit. The other instructions follow this, except where they are immediately describing what those bits signify.

Reply:
It was AAA before, too, so 🤔 fixed